docs/references/expo/local-credentials.mdx

```diff
@@ -0,0 +1,235 @@
+---
+description: Learn how to use local authentication in your Expo app with Clerk.
+---
+
+# Use biometrics with Local Credentials
```

Comment:
I'm confused about the purpose of this guide.
There's three things going on here: local auth, biometrics, and NOT signing in, but re-signing in ??

What is the purpose of this guide?

Reply:
We are adding a new feature in Expo, where the app can allow returning users to use Face ID instead of identifier + password.

Local Authentication is the term used by the native platform to describe getting access to the biometrics of the device like Face ID, or Touch ID.